index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+  <script src="index.js"></script>
+  <link rel="stylesheet" href="index.css">
+  <title>Document</title>
+</head>
+<body>
+  <header id="header">
+    <img src="https://upload.wikimedia.org/wikipedia/en/thumb/c/ca/Studio_Ghibli_logo.svg/1200px-Studio_Ghibli_logo.svg.png" alt="Ghibli logo">
+    <h1>Ghibi Review App</h1>
+  </header>
+
+  <main id="mainArea">
+    <select name="" id="">
+    <option value="" disabled selected>Select a Movie</option>
+  </select>
+  </main>
+
+ <info id="displayInfo">
+   <h3 id="title"></h3>
+   <p id="year"></p>
+   <p id="description"></p>
+ </info>
+ 
+ <form action="">
+   <input type="text">
+   <button id="submit">Submit</button>
+ </form>
+
+ <reviewContainer id="review">
```

Comment:
Although this is valid, it would better just to create a `div` and give it the id of review